data/json/recipes/recipe_medsandchemicals.json

```diff
@@ -798,6 +798,68 @@
   ]
 },{
   "type" : "recipe",
+  "result": "mutagen_lamia",
+  "category": "CC_CHEM",
+  "subcategory": "CSC_CHEM_MUTAGEN",
+  "skill_used": "cooking",
+  "skills_required": [ "firstaid", 1 ],
+  "difficulty": 9,
+  "time": 10000,
+  "reversible": false,
+  "autolearn": false,
```

Comment:
If it's redundant here, it's redundant across the entirety of all of the mutation recipes. Should I then remove them from all of the other recipes while I'm at it, if that'll optimize the code a little more?